### ISSUE TITLE
How to make "How to install Ubuntu" easier for the average computer user

### ISSUE THREAD

**Issue by josephernest** [canonical-websites/www.ubuntu.com#3440]
I'd like to share my experience of giving instructions by phone about *"How to install Ubuntu"* to my nephew.

*Context: I have installed various Linux distros hundreds of times myself in my life, so no problem for me, but when showing it / giving instructions to someone unexperienced, I have noticed  some small things that could be improved on the website, easy to fix, but that would make it a loooooooot easier "for the average Joe".*

### 1. For an unexperienced user, the "home page" does not exactly provide an easy access to *"Install it on your computer".*

Ok **we** do know we have to go *"Desktop > Overview"* and then locate the "Download Ubuntu" button, etc.

![](https://i.imgur.com/m1dy2Lx.jpg)

But for the average user, **the home page** (and even the top of the page (visible in the viewport) **should feature** a "I want to install Ubuntu on my computer". Something simple.
Finding an option in a submenu is too complex. 

If it was the *only step* to do, it wouldn't be too complex, but given the complexity of the next steps (see next paragraphs), the fact the home page doesn't provide easy access to what people want (usually: install Ubuntu!) *on the first visible viewport of the brower* is not so great in terms of user experience.

### 2. The path to Rufus (on Windows)

When the ISO is downloaded on https://www.ubuntu.com/download/desktop, the average user **is lost**. Totally **lost**. Of course **we** do know we have to find **this little link** on the **bottom of the page**: "How to create a bootable USB stick on Windows". 

But the average user doesn't.

Once clicked he sees a long tutorial (9 minutes written on top right): https://tutorials.ubuntu.com/tutorial/tutorial-create-a-usb-stick-on-windows

Step number 3 eventually brings to https://rufus.akeo.ie/. This page FULL OF TEXT is hard to parse for a non-power user. Many tech things. Ok I do know in 0.5 sec that I should just click on "Rufus 3.1 Portable (995 KB)" download link. But the non-power user doesn't necessarily know this:

![Imgur](https://i.imgur.com/aoE3Vlw.jpg)

### 3. The Rufus journey

![](https://rufus.akeo.ie/pics/rufus_en.png)

A little cup of MBR? Or maybe do you want some BIOS or UEFI? What's your favorite cluster size? 4096 or maybe 8192?
Ok, *we* do know what to do about this. But my uncle (who is not a noob, he's a computer enthusiast since decades) doesn't know what is a MBR. My mother also doesn't. My nephew also doesn't (yet).

Do we really want them out of the Ubuntu world??

### 4. Suggested solution

Please do for example like Mozilla: https://www.mozilla.org/en-US/firefox

Example:

The website can detect if user has Windows. 
If so, instead of redirecting him to Rufus, please build your own bootable USB maker tool with nearly no other option than "Choose your USB stick" and "Choose your ISO" and "Go!".
There are lots of such open-source freeware tool. Canonical could easily build your own (it will take 2 days for an intern) "UbuntuBootableUSBMaker.exe".

Quick mockup:

![Imgur](https://i.imgur.com/CGKtRYD.jpg)

So the user only has to:

* Download the ISO (~ 2GB)
* Download the Ubuntu Bootable USB maker tool (~ 1 Mo), a little like the Mozilla net installer (~ 1MB), that everyone downloads when installing Firefox.

Thanks for having read my 2 cents :)

This would really make life easier for many people in the path to Ubuntu :)

**Comment by anthonydillon** [comment]
Could a UXer please consider these suggestions and create subtasks to cover the outcomes, please.

**Comment by josephernest** [comment]
Thanks for your support @anthonydillon. 

Funnily, 5 minutes after submitting this post in June, my girlfriend called me "I think it's today", and one hour later we were at maternity hospital. We could have called our daughter "Ubuntu" :)

**Comment by pmahnke** [comment]
See #1283
